### PR TITLE
E2e testing - fix failing test

### PR DESCRIPTION
**Pull request by PJSalter**
- removed old test code that was failing.
- wrote pseudo code for clear steps to follow.
- broke it down by passing each step.
- ran single test by specifying test name `inputText.spec.ts` after `npx playwright test`
- looped through classname to show multiple elements of jokes and  puns.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8707,6 +8707,13 @@ react-element-to-jsx-string@^14.3.4:
     is-plain-object "5.0.0"
     react-is "17.0.2"
 
+react-error-boundary@^4.0.3:
```
Comment (millipede-cpu) [inline yarn.lock:8710]: Thank you for running yarn, I must've missed doing that when reviewing your previous branch errorHandle

File: tests/inputText.spec.ts [hunk 2]
```diff
@@ -1,28 +1,39 @@
-import { chromium, test } from "@playwright/test";
+import { chromium, expect, test } from "@playwright/test";
 
-test("Typing a joke and pun should only appear on the page after clicking the Submit Joke button", async () => {
+test("Typing a joke and a pun and clicking the seperate buttons should render the text on the homepage", async () => {
```
Comment (millipede-cpu) [inline tests/inputText.spec.ts:3]: Much clearer description and pseudocode is very helpful to break down each step of the test as it passes 🥳

File: tests/inputText.spec.ts [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
   await page
-    .locator("#joke", {
-      hasText: "Why do scientists not trust atoms?",
-    })
-    .locator(".submit-joke")
-    .click({ force: true });
-  await page.locator(
-    '.showJokePun:has-text("Why do scientists not trust atoms?")'
-  );
+    .getByPlaceholder("Type something funny")
```
Comment (PJSalter) [inline tests/inputText.spec.ts:15]: This is brilliantly neat testing that is clearly understandable with all steps we have made, we have done a great job

**Review by millipede-cpu — APPROVED** [review]

File: tests/inputText.spec.ts [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
-    .locator(".submit-pun")
-    .click({ force: true });
-  await page.locator(
-    '.showJokePun:has-text("Because they make up everything.")'
-  );
+    .locator(
+      '.showJokePun:has-text("I used to be addicted to soap, but I\'m clean now.")'
+    )
+    .isVisible();
+  // 5. user inputs text with placeholder whats the pun?
```
Comment (PJSalter) [inline tests/inputText.spec.ts:25]: clear and neat steps 🧪

**Pull request merged**